eth-contracts/contracts/Governance.sol

```diff
@@ -0,0 +1,294 @@
+pragma solidity ^0.5.0;
+
+import "./service/registry/RegistryContract.sol";
+import "./staking/Staking.sol";
+import "./service/interface/registry/RegistryInterface.sol";
+
+contract Governance {
+  RegistryInterface registry = RegistryInterface(0);
+  bytes32 stakingProxyOwnerKey;
+
+  // 48hr * 60 min/hr * 60 sec/min / ~15 sec/block = 11520 blocks
+  uint256 votingPeriod = 11520;
+
+  uint256 votingQuorum = 1;
+
+  /***** Enums *****/
+  enum Outcome {InProgress, No, Yes, Invalid}
+  // Enum values map to uints, so first value in Enum always is 0.
+  enum Vote {None, No, Yes}
+
+  struct SlashProposal {
+    uint256 proposalId;
+    address proposer;
+    uint256 startBlockNumber;
+    address target;
```

Comment:
platforms will often have a string or associated text with each slashProposal describing _why_ a slash action is happening. how do we plan on handling this? personally i do think a short (like < 256 char) description on chain is very reasonable or even an event - but having nothing is an odd dynamic, and it **cannot** be in something like identity

Reply:
i was thinking about this too. i think most other protocols have either a github page or custom dashboard url for each proposal. could we require a url to that?

Reply:
100% agreed on this, don't think it's blocker for v0 but added a card to notion checklist and will do as fast follow

Reply:
including this in v1 pr